Added a more basic example as the template ones are tough to use as s… (#14223)
home-assistant/home-assistant.github.io: source/_integrations/rest_command.markdown

## Examples

### Basic example which uses PUT method and payload encoded as form data

> This example implements 2 rest commands to add service calls the missing shuffle functionality from the iTunes integration with a REST command.

Reviewer: ```suggestion
This example implements 2 REST commands to add service calls for the missing shuffle functionality of the iTunes integration.
```

```yaml
rest_command:
  shuffle_on: 

>     url: "http://[YOUR_ITUNES-API_SERVER_IP]:8181/shuffle"

Reviewer: ```suggestion
    url: "http://YOUR_ITUNES-API_SERVER_IP:8181/shuffle"
```

    method: put
    content_type: "application/x-www-form-urlencoded"
    payload: "mode=songs"
  shuffle_off: 

>     url: "http://[YOUR_ITUNES-API_SERVER_IP]:8181/shuffle"

Reviewer: ```suggestion
    url: "http://YOUR_ITUNES-API_SERVER_IP:8181/shuffle"
```

    method: put
    content_type: "application/x-www-form-urlencoded"
    payload: "mode=off"
```

### Using templates to change the payload based on entities

The commands can be dynamic, using templates to insert values of other entities. Service call support variables for doing things with templates.

> In this example entry, you can see some simple [templates](/docs/configuration/templating/) in use for dynamic parameters.

Reviewer: ```suggestion
In this example, uses [templates](/docs/configuration/templating/) for dynamic parameters.
```
